OpenRA.Game/GameRules/MusicInfo.cs

```diff
@@ -36,11 +35,11 @@ public MusicInfo(string key, MiniYaml value)
 
 		public void Load()
 		{
-			if (!GlobalFileSystem.Exists(Filename))
+			if (!Game.ModData.ModFiles.Exists(Filename))
```

Comment:
As for the packages, it would be better if this referenced a passed `ModData` (as opposed to `ModFiles`, which feels a little bit too specific IMO) so that we can unstatic this now instead of in a future pass.